### PR TITLE
Add flag option to target a directory to run/build/deploy a function

### PR DESCRIPTION
**Pull request by JadeRedworth**
As a developer I would like to run Fn CLI commands providing a target working directory [#268](https://github.com/fnproject/cli/issues/268)

### CODE REVIEW

File: commands/init.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func initFlags(a *initFnCmd) []cli.Flag {
 			Destination: &a.ff.Version,
 			Value:       common.InitialVersion,
 		},
+		cli.StringFlag{
+			Name:  "dir",
+			Usage: "specify directory to build a function",
```
Comment (rikgibson) [inline commands/init.go]: the text here should say "init a function" rather than "build"...

File: common/bump.go [hunk 2]
```diff
@@ -63,12 +62,17 @@ func (b *bumpcmd) flags() []cli.Flag {
 			Usage:       "verbose mode",
 			Destination: &b.verbose,
 		},
+		cli.StringFlag{
+			Name:  "dir",
+			Usage: "specify directory to deploy a function",
```
Comment (rikgibson) [inline common/bump.go]: "bump" not "deploy"

**Comment by shaunsmith** [comment]
@rikgibson I agree with the scenario 1-4 semantics you described.  Are you saying the current changes don't align with these semantics or that we need to capture this in help text?

I agree that `dir` is working directory but don't think we need verbose switch names.  `--dir` and `-d` are nice and succinct and the semantics should be covered in the help text.

**Comment by JadeRedworth** [comment]
@shaunsmith These current changes don't align. Rik and I had a catch-up today and I had misinterpreted the use of --dir within the init command. We are on the same page now and I will make the suggested changes.

**Pull request merged**